packages/rocketchat-language-plugin/client/client.js

```diff
@@ -0,0 +1,41 @@
+import toastr from 'toastr';
```

Comment:
I am not sure we want to use toastr for notificatios. I think a better approach would be just to add users to channels, and when they open a channel they have information about the fact that this is automatically created channel for them.